patches/node/src_use_cp_utf8_for_wide_file_names_on_win32.patch

```diff
@@ -136,10 +136,21 @@ index 969e7d08086f8442bed476feaf15599b8c79db7c..e7459654401c275dfb86207831016ed7
                      std::make_error_code(std::errc::file_exists),
                      "cp",
 diff --git a/src/node_modules.cc b/src/node_modules.cc
-index d8477191efafba3c41c06d765f4b03bd00b8573c..5444e556b89ba5b71739753eaafa706cd9727013 100644
+index d8477191efafba3c41c06d765f4b03bd00b8573c..3b4d82da2ad30cafa96611eaa2d68ddf1badeac0 100644
 --- a/src/node_modules.cc
 +++ b/src/node_modules.cc
-@@ -365,12 +365,13 @@ const BindingData::PackageConfig* BindingData::TraverseParent(
+@@ -352,8 +352,8 @@ const BindingData::PackageConfig* BindingData::TraverseParent(
+         });
+   };
+ 
+-  bool did_original_path_start_with_resources_path = starts_with(check_path.
+-      generic_string(), resources_path);
++  bool did_original_path_start_with_resources_path = starts_with(
++      ConvertGenericPathToUTF8(check_path), resources_path);
```

Comment:
Hm... I might actually need to look into how `resources_path` computed when strings has unicode in it.